### ISSUE TITLE
Bad /etc/sudo.conf permissions, other issues?

### ISSUE THREAD

**Issue by scop** [hassio-addons/addon-ssh#111]
# Problem/Motivation

> (Why the issue was filed)

On first login to a 7.0.0 addon, with just authorized key added and zsh set to false, I'm greeted with these errors before the prompt:
```
sudo: /etc/sudo.conf is world writable
sudo: /etc/sudo.conf is world writable
sudo: setrlimit(RLIMIT_CORE): Operation not permitted
```

## Expected behavior

> (What you expected to happen)

No errors.

## Actual behavior

> (What actually happened)

See above.

## Steps to reproduce

> (How can someone else make/see it happen)

Clean 7.0.0 install, add authorized key, set zsh to false, log in.

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

**Comment by frenck** [comment]
So weird, it isn't showing that on my end:

![image](https://user-images.githubusercontent.com/195327/71581245-9d786c80-2b04-11ea-9efd-f1d857fb1e49.png)

Nevertheless, I'll make an adjustment to ensure the permissions.

**Comment by frenck** [comment]
Addressed in v7.0.1.

Thanks for reporting this in @scop!
